### PR TITLE
Update text in video-contenthint.

### PR DESCRIPTION
**Pull request by pbos**
Fixes a grammar error and rephrases the paragraph on what setting
MediaStreamTrack content hints does.

### CODE REVIEW

File: src/content/capture/video-contenthint/index.html [hunk 1]
```diff
@@ -57,11 +57,11 @@
...
+    <p>The transmitted stream tracks are using <a href="https://wicg.github.io/mst-content-hint/">MediaStreamTrack Content Hints</a> to indicate characteristics in the video stream, which informs PeerConnection on how to encode the contained stream (to prefer fluid motion or individual frame detail).</p>
 
-    <p>The text part of the clip shows a clear case for when <tt>'detailed'</tt> is better, and the fighting scene shows a clear case for when <tt>'fluid'</tt> is better. The spinning model however shows a case where <tt>'fluid'</tt> or <tt>'detailed'</tt> are not clear-cut decisions and what's preferred is here up to the user, even with good content detection.</p>
+    <p>The text part of the clip shows a clear case for when <tt>'detailed'</tt> is better, and the fighting scene shows a clear case for when <tt>'fluid'</tt> is better. The spinning model however shows a case where <tt>'fluid'</tt> or <tt>'detailed'</tt> are not clear-cut decisions and even with good content detection what's preferred is here up to the user.</p>
```
Comment (alvestrand) [inline src/content/capture/video-contenthint/index.html]: grammar: "is here up to the user" -> "is up to the user in this case", or simpler: "depends on what the user prefers".

File: src/content/capture/video-contenthint/index.html [hunk 2]
```diff
@@ -57,11 +57,11 @@
...
 
-    <p>Other MediaStreamTrack consumers such as MediaStreamRecorder can also make use of this decision to guide encoding parameters for the stream without additional extensions to the MediaStreamRecorder specification, but they are currently not implemented.</p>
+    <p>Other MediaStreamTrack consumers such as MediaStreamRecorder can also make use of this decision to guide encoding parameters for the stream without additional extensions to the MediaStreamRecorder specification, but they are currently not implemented in Chromium.</p>
```
Comment (alvestrand) [inline src/content/capture/video-contenthint/index.html]: "this decision" -> "this information"
"they are currently not implemented" -> "this is currently not implemented".

**Review by alvestrand — APPROVED** [review]
Looks like an improvement, grammar nits.

File: src/content/capture/video-contenthint/index.html (hunk 1 above)
Comment (pbos) [reply src/content/capture/video-contenthint/index.html]: Done, thanks!

File: src/content/capture/video-contenthint/index.html (hunk 2 above)
Comment (pbos) [reply src/content/capture/video-contenthint/index.html]: Done!

**Pull request merged**